common/src/test/java/com/alibaba/nacos/common/utils/JacksonUtilsTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *   Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package com.alibaba.nacos.common.utils;
```

Comment:
It is not in client.security. Are you sure commit in this PR?

Reply:
these code was put in wrong place in package client.security ,I just rename  it here. Or I should remove it

Reply:
These two package has much different, even different module. So I think split it to two PR will be better

Reply:
done